index.html

```diff
@@ -4,22 +4,29 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>One of Saturn's largest rings may be newer htan anyone</title>
-    <link rel="stylesheet" href="Style.css">
+    <title>One of Saturn's largest rings may be newer than anyone</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
     <header class="clearfix">
-        <div class="logotype">FASHION</div>
+        <div class="container">
+            <div class="header-logo">
+                <a href="" class="header-logo">
+                    <img src="/Users/igorgryshajev/Documents/Various/Figma-IG/imgs/Logotype.svg">
```

Comment:
path has to be relative. On a different machine it's gonna be different

```suggestion
                    <img src="./imgs/Logotype.svg">
```